scarb/src/bin/scarb/args.rs

```diff
@@ -132,12 +130,22 @@ pub enum Command {
     Metadata(MetadataArgs),
     /// Create a new Scarb package at <PATH>.
     New(NewArgs),
+    /// Run arbitrary package scripts.
+    Run(ScriptsRunnerArgs),
 
     /// External command (`scarb-*` executable).
     #[command(external_subcommand)]
     External(Vec<OsString>),
 }
 
+#[derive(Parser, Clone, Debug)]
+#[clap(trailing_var_arg = true)]
+pub struct ScriptsRunnerArgs {
```

Comment:
how did this survive `#[deny(missing_docs)]` ??? 😱

Reply:
😱